src/platforms/esp32/main/main.c

```diff
@@ -136,3 +149,33 @@ void app_main()
         }
     }
 }
+
+#ifdef USE_USB_SERIAL
+void init_usb_serial()
+{
+    /* Setting TinyUSB up */
+    ESP_LOGI(TAG, "USB initialization");
+
+    const tinyusb_config_t tusb_cfg = {
+        .device_descriptor = NULL,
+        .string_descriptor = NULL,
+        .external_phy = false, // In the most cases you need to use a `false` value
+#if (TUD_OPT_HIGH_SPEED)
+        .fs_configuration_descriptor = NULL,
+        .hs_configuration_descriptor = NULL,
+        .qualifier_descriptor = NULL,
+#else
+        .configuration_descriptor = NULL,
+#endif // TUD_OPT_HIGH_SPEED
+    };
+
+    ESP_ERROR_CHECK(tinyusb_driver_install(&tusb_cfg));
+
+    tinyusb_config_cdcacm_t acm_cfg = { 0 }; // the configuration uses default values
+    ESP_ERROR_CHECK(tusb_cdc_acm_init(&acm_cfg));
+
+    ESP_LOGI(TAG, "USB initialization DONE");
```

Comment:
Should this be logged after `esp_tusb_init_console` so that the output is displayed?